db/db_impl/db_impl.h

```diff
@@ -1281,6 +1281,17 @@ class DBImpl : public DB {
       ColumnFamilyData* cfd, const ExternalSstFileIngestionJob& ingestion_job);
 
   Status FlushForGetLiveFiles();
+
+  Status IsFlushMemtableNeededForGetLiveFiles(
+      const LiveFilesStorageInfoOptions& opts, bool& need_flush_memtable);
+  void GetLiveFilesExceptWals(const LiveFilesStorageInfoOptions& opts,
+                              std::vector<LiveFileStorageInfo>& results,
```

Comment:
I, and IIRC the rest of the team with opinions, prefer the old Google style that did not allow non-const reference parameters. Why? Readability. (I shouldn't have to know the signatures of called functions to construct reasonably accurate UD/DU chains on local variables - https://en.wikipedia.org/wiki/Use-define_chain)

Reply:
Actually, team may not agree, but here are some references about old style. Not sure why it changed.

https://stackoverflow.com/questions/26441220/googles-style-guide-about-input-output-parameters-as-pointers
https://www.youtube.com/watch?v=NOCElcMcFik (Seek to 15:45 and to 28:28)
https://github.com/google/styleguide/commit/7a7a2f510efe7d7fc#diff-bcadcf8be931ffdd5d6a65c60c266039cf1f96b7f35bfb772662db811214c5a0R1713